core/tools.js

```diff
@@ -1596,6 +1596,64 @@
 			return base64string;
 		},
 
+		/**
+		 * Converts blob url into base64 string. Function requires {@link CKEditor.ajax} to proper work. Conversion has happened asynchronously.
+		 * Currently supported file types: `image/png`, `image/jpeg`, `image/gif`.
+		 *
+		 * @since 4.10.0
+		 * @param {String} blobUrlSrc Address of blob which is going to be converted
+		 * @param {Function} callback Function execute when blob url will be converted.
+		 * @param {String} callback.dataUri data uri represent transformed blobUrl or empty string file type was unrecognized.
+		 */
+
+		convertBlobUrlToBase64: function( blobUrlSrc, callback ) {
+			CKEDITOR.ajax.load( blobUrlSrc, function( arrayBuffer ) {
```

Comment:
`ajax` plugin function can not be used in core, since it's not guaranteed it's going to be there. If anything the logic should go to the ajax plugin itself.